internal/pkg/util/fs/fuse/fuse_mount_linux.go

```diff
@@ -43,11 +44,15 @@ type ImageMount struct {
 
 	// AllowOther is set to true to mount the image with the "allow_other" option.
 	AllowOther bool
+
+	// ExtraMountOpts are options to be passed to the mount command (in the "-o"
+	// argument) beyond the ones autogenerated from other ImageMount fields.
+	ExtraMountOpts []string
```

Comment:
Either `Opts` or `ExtraOpts`. Mount is redundant given this is `ImageMount.xxxx`